src/zenml/client.py

```diff
@@ -5391,11 +5391,15 @@ def _get_entity_by_id_or_name_or_prefix(
 
         # If more than one entity with the same name is found, raise an error.
         entity_label = get_method.__name__.replace("get_", "") + "s"
+        formatted_entity_items = [
+            f"- {item.name}: (id: {item.id}, flavor: {item.flavor})\n"
```

Comment:
This won't work since only stack components have a `.flavor` field, but this util function is used for many other entities as well

Reply:
Updated to address that. Do you think we should conditionally output that if they are stack components? Or anything else you'd think we might want to include in the output here that's present for all entities?